Multisig/Logic/Models/AppSettings.swift

```diff
@@ -151,7 +148,6 @@ struct PasscodeOptions: OptionSet {
     static let useBiometry = PasscodeOptions(rawValue: 1 << 0)
     static let useForLogin = PasscodeOptions(rawValue: 1 << 1)
     static let useForConfirmation = PasscodeOptions(rawValue: 1 << 2)
-    static let useForExportingKeys = PasscodeOptions(rawValue: 1 << 3)
 
-    static let all: PasscodeOptions = [.useBiometry, .useForLogin, .useForConfirmation, .useForExportingKeys]
```

Comment:
passcode option values are confusing here. It makes impression that key export is not protected by passcode anymore.